ffmpeg-plugin/mcm_audio_tx.c

```diff
@@ -28,33 +28,21 @@ typedef struct McmAudioMuxerContext {
     int sample_rate;
     char* ptime;
 
-    mcm_conn_context *tx_handle;
-    mcm_buffer *unsent_mcm_buf;
+    MeshClient *mc;
+    MeshConnection *conn;
+    MeshBuffer *unsent_buf;
     int unsent_len;
+
```

Comment:
empty line not needed

Reply:
Fixed in eae8b083e0210567994eb379b5e228742451c44b.